playbooks/roles/common-ses/tasks/main.yml

```diff
@@ -0,0 +1,38 @@
+---
+- name: Include global vars
+  include_vars: "{{ playbook_dir }}/../vars/common-vars.yml"
+
+- name: Create pools in SES for socok8s usage
+  become: yes
+  include_role:
+    name: ses
+    tasks_from: openstack_config
+  vars:
+    ses_openstack_config: True
+    ses_openstack_glance_pool:
+      name: socok8s_images
+      pg_num: "{{ ses_osd_pool_default_pg_num }}"
+    ses_openstack_cinder_pool:
+      name: socok8s_volumes
+      pg_num: "{{ ses_osd_pool_default_pg_num }}"
+    ses_openstack_nova_pool:
+      name: socok8s_vms
+      pg_num: "{{ ses_osd_pool_default_pg_num }}"
+    ses_osh_rbd_pool:
+      name: socok8s_rbd
+      pg_num: "{{ ses_osd_pool_default_pg_num }}"
+    ses_openstack_cinder_backup_pool:
+      name: socok8s_cinder_backup
+      pg_num: "{{ ses_osd_pool_default_pg_num }}"
+    ses_openstack_pools:
+      - "{{ ses_openstack_glance_pool }}"
+      - "{{ ses_openstack_cinder_pool }}"
+      - "{{ ses_openstack_nova_pool }}"
+      - "{{ ses_openstack_cinder_backup_pool }}"
+      - "{{ ses_osh_rbd_pool }}"
+
+- name: Import SES configs to deployer
+  become: yes
+  include_role:
+    name: ses-osh
```

Comment:
eventually, we should have the code in the commons, and have ses-osh to import from commons, since ses-osh is dev mechanism, IMO.

Reply:
Yes, makes sense. So will push update to PR to follow this usage model. Actually I can remove usage of ses-osh role altogether and directly invoke play from role common-ses.

Reply:
The requested change was made in susequent PR push so resolving this now